front-end/src/app/components/serverlist/ServerList.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import './ServerList.scss';
+import {Button} from '@material-ui/core';
+import {Link} from 'react-router-dom';
+import 'material-design-icons/iconfont/material-icons.css';
+
+import star from '../../../assets/images/star.png';
+import map from '../../../assets/images/sky.png';
+
+export default class ServerListComponent extends React.Component {
+
+  public render() {
+
+    const items = [];
+    // items est temporaire et sera à remplacer par la liste des serveurs avec la map en cours et l'avatar des joueurs
+    // Il faut garder les className pour le SCSS
```

Comment:
Everything in english please! Also className is the equivalent of class in the HTML... so this comment is rather useless.